new-frontiers-14/frontier-station-14
# Fixes the Admeme Teleporter Despawns

## About the PR


Makes a new Portal entity as `PortalArtifactNoDespawn` That is basicly the same as `PortalArtifact` but without the `TimedDespawn` component

I do not know if theres an better way to compress the YML, but it does its job.

## Why / Balance

The portal would disappear after 120 seconds, which is bad for the admeme teleporter as it is annoying to replace the portal every 120 seconds.

## How to test

Run fork
Spawn admeme Teleporter
Setup portals
See it not despawn after some time

## Media



## Breaking changes


**Changelog**

No changelog needed.

:cl:
- fix: Admin portals no longer despawn.
